.github/workflows/test1.yml

```diff
@@ -19,6 +23,8 @@ on:
       - 'metrics/**'
       - 'docs/**'
       - '**.md'
+      - '!waku/v1/**.nim'
+      - '!tests/v1/**.nim'
```

Comment:
This was deliberate. We want all tests to be executed when merging into `master`. Please, remove this too.